### DISCUSSION TITLE
When set public, query not found

### DISCUSSION THREAD

**Discussion by jasulainen** [Budibase/budibase#3414] [Help]
I have simple REST API application that would just list stuff from the API.

When i'm logged in to the budibase, everything looks fine (whichever way i navigate to the app). Application screens are set "public".

When I try the same in incognito mode page loads ok but it gives error "query not found". 

I tried to find some place where to setup some additional access control or something regarding queries but haven't found any.

![image](https://user-images.githubusercontent.com/383746/142224980-e72afe7d-f15f-4269-9761-8d8bf1643556.png)

**Comment by mjashanks** [comment]
Hi There. 

We currently cannot change the access level of a custom query. So, it cannot be made public.

We have an issue logged for this: https://github.com/Budibase/budibase/issues/2817
